lib/convert_json_schema.rb

```diff
@@ -163,6 +170,10 @@ def convert_to_json_schema(props, parent)
         v['enum'].delete('video/generation')
       end
 
+      if ['minLength', 'minItems', 'minimum'].include?(k) && v < 1
```

Comment:
isn't `minimum` a valid json-schema property?

Reply:
It removes them if the value is < 1, otherwise it keeps them